src/autocomplete/style.js

```diff
@@ -0,0 +1,26 @@
+import styled, { css } from 'styled-components';
+
+const DarkBg = css`
+  background-color: #f0f0f8;
+`;
+
+export const Container = styled.div``;
```

Comment:
Maybe I'm being pedantic but I'm not sure I see the value in creating a styled component with no styles?